tests/library/client-certificates.spec.ts

```diff
@@ -56,6 +56,15 @@ const test = base.extend<{ serverURL: string, serverURLRewrittenToLocalhost: str
   }
 });
 
+test.use({
```

Comment:
We'll remove that again once https://github.com/microsoft/playwright/issues/17252 landed after Chromium 129.